frontend/src/components/AccountSendTable.vue

```diff
@@ -0,0 +1,150 @@
+<template>
```

Comment:
How much of this is similar to the `AccountReceiveTable`? Does it make sense to extract a `BaseSendsTable` component and reuse that in the `AccountReceiveTable` and `AccountSentTable` components?

Reply:
I think it is similar but a pared down version. There are probably some things that can be abstracted

Reply:
Should we track that in a separate issue or do it here? Hmm or maybe it's not worth abstracting since it's only 2 instances 🤔

Reply:
I think a separate issue would make sense. And we abstract when the next use case comes